editor.planx.uk/src/@planx/components/MapAndLabel/test/mocks/Trees.ts

```diff
@@ -20,3 +24,13 @@ export const props: PresentationalProps = {
   longitude: -0.1629784,
   latitude: 51.5230919,
 };
+
+export const previousSingleDataProps: PresentationalProps = {
+  ...props,
+  previouslySubmittedData: previouslySubmittedSingleData,
```

Comment:
The bug was different if there was a single data point added or if there was more than one, thus I added two separate bug tests